src/Components/Footer/Style.jsx

```diff
@@ -0,0 +1,325 @@
+import styled from 'styled-components';
+import { FacebookSquare } from '@styled-icons/boxicons-logos/FacebookSquare';
+import { Instagram } from '@styled-icons/boxicons-logos/Instagram';
+import { Twitter } from '@styled-icons/entypo-social/Twitter';
+import { LinkedinSquare } from '@styled-icons/boxicons-logos/LinkedinSquare';
+
+const SFooter = styled.footer`
+  position: absolute;
+  bottom: 0;
+  background-color: var(--color--secondary);
+  width: 100vw;
+  height: 30vh;
+  color: var(--color--thirdary);
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  @media (max-width: 768px) {
+    height: 80vh;
+  }
+`;
+
+const FcbIcon = styled(FacebookSquare)`
```

Comment:
I think it would be best to create a generic style and apply it to your icons. You will save a lot of lines of code